src/lib/utils/column_ids_after_pruning.cpp

```diff
@@ -0,0 +1,33 @@
+#include "column_ids_after_pruning.hpp"
+
+#include <cstdlib>
+#include <optional>
+#include <vector>
+
+namespace opossum {
+
+const std::vector<std::optional<ColumnID>> column_ids_after_pruning(const size_t& original_table_column_count,
+                                                                    const std::vector<ColumnID>& pruned_column_ids) {
+  std::vector<std::optional<ColumnID>> column_id_mapping;
+
+  auto column_pruned_bitvector = std::vector<bool>();
```

Comment:
Is there any reason for not creating the vector as the one above? In addition, you could avoid the resizing by passing `original_table_column_count` to the constructor.

Reply:
Good point, no specific reason.